dali/operators/reader/video_reader_op.h

```diff
@@ -168,18 +155,23 @@ class VideoReader : public DataReader<GPUBackend, SequenceWrapper> {
 
   void RunImpl(DeviceWorkspace &ws) override {
     auto& video_output = ws.Output<GPUBackend>(0);
+    auto& curent_batch = prefetched_batch_tensors_[curr_batch_consumer_];
 
-    PrepareVideoOutput(video_output, ws);
+    SetOutputShapeType(video_output, ws);
     PrepareAdditionalOutputs(ws);
 
-    for (int data_idx = 0; data_idx < batch_size_; ++data_idx) {
-      auto& prefetched_video = GetSample(data_idx);
+    ProcessVideo(video_output, curent_batch, ws);
+    video_output.SetLayout("FHWC");
```

Comment:
maybe setting layout should be part of SetOutputShapeType?

Reply:
Done